tests/test_config_consistency.py

```diff
@@ -71,3 +71,39 @@ def test_status_code_202(self):
         assert spans[0]["type"] == "web"
         assert spans[0]["meta"]["http.status_code"] == "202"
         assert spans[0]["error"] == 1
+
+@scenarios.tracing_config_nondefault
+@features.tracing_configuration_consistency
+class Test_Config_IntegrationEnabled_False:
+    """ Verify behavior of integrations automatic spans """
+
+    def setup_mongodb_integration_enabled_false(self):
+        self.r = weblog.get("/integration_enabled_config")
```

Comment:
Two small comments here:
1. Can you rename this to something more explicit about what the endpoint is doing, like `/mongodb/getdatabase`?
2. Can you document the endpoint in `docs/weblog/README.md`?

Reply:
Addressed in: https://github.com/DataDog/system-tests/pull/3060/commits/957ea233b8fd4a3bdd64a6294a241eab6fb9f8eb